python/port/genhdr/qstrdefs.in.h

```diff
@@ -266,22 +263,20 @@ Q(lstrip)
 Q(map)
 Q(math)
 Q(matplotlib)
-Q(matplotlib.pyplot)
+Q(matplotlib_dot_pyplot)
```

Comment:
```suggestion
Q(matplotlib.pyplot)
```

Reply:
I'll try to compile this and see if it works. Yesterday I tested matplotlib and it didn't exist as a module, so I hope this fixes it.

Reply:
I already try to upgrade it, but I have an attribute error… #135

Reply:
I didn't see your PR, my bad. I suppose I'll look into fixing that, if I can. It's either that, or we wait for Epsilon to actually update to 1.18 and see how they do it.